test/e2e/common.go

```diff
@@ -273,7 +274,7 @@ func (e2e *ZarfE2ETest) checkIfClusterRunning() bool {
 
 func (e2e *ZarfE2ETest) waitForHealthyCluster() error {
 	attempt := 0
-	for attempt < 10 {
+	for attempt < 15 {
```

Comment:
any details on why this change was needed?  Nbd, just curuious

Reply:
I noticed a k3d test failed once because the cluster didn't have enough time to be healthy. It's somewhat a shot in the dark but I figured it wouldn't hurt to give the runners a few extra seconds to spinup the cluster since we don't actually know the underlying specs of the hardware.